drivers/modem/Kconfig.ublox-sara-r4

```diff
@@ -17,6 +17,12 @@ config MODEM_UBLOX_SARA_R4
 
 if MODEM_UBLOX_SARA_R4
 
+config MODEM_UBLOX_SARA_U2
+	bool "u-blox SARA-U2"
+	default n
```

Comment:
Can skip the `default n`. See the [redundant defaults](https://docs.zephyrproject.org/latest/guides/kconfig/index.html#redundant-defaults) section of the Kconfig tips page.